Development/nmos/sdp_attributes.h

```diff
@@ -0,0 +1,48 @@
+#ifndef NMOS_SDP_ATTRIBUTES_H
+#define NMOS_SDP_ATTRIBUTES_H
+
+#include "sdp/json.h"
+
+namespace nmos
+{
+    namespace sdp_attributes
+    {
+        // RTP Header Extensions
+        // See https://tools.ietf.org/html/rfc5285#section-5
+        struct extmap
+        {
+            uint64_t local_id;
+            sdp::direction direction;
+            utility::string_t uri;
```

Comment:
@N-Nagorny What do you think about making this a `web::uri`? Would that be good or bad? It would mean `make_extmap()` could throw `web::uri_exception`, and I'm not sure that's what we want. (We have lots of "typed string enums" in e.g. `nmos::sdp_parameters` but these don't throw if constructed with a string that isn't a recognized value.)

Reply:
I don't see how `web::uri` can help for constructing an entry, it's too permissive. It would be useful for parsing but I don't know if it makes sense if `scheme` usually returns `"urn"` and `path` returns the rest of the string and the other functions return nothing. Let's leave it as `utility::string_t`.